### ISSUE TITLE
Correct me if I'm wrong

### ISSUE THREAD

**Issue by pjebs** [mikenikles/deno-sapper-cloud-run#3]
So this proof of concept runs sapper on a deno server provided the project is static website?

Can I still call Ajax requests to a go backend?

**Comment by mikenikles** [comment]
Yes, it only supports a static export of a Sapper app. You can make Ajax requests to any URL as you usually would, there's no change to that 👍.

Do you run into any specific issues? If so, feel free to post it here and I'll see what I can do to help you out.

**Comment by pjebs** [comment]
Does this mean I can create a Sapper application with no node backend (and not SSR)?

**Comment by pjebs** [comment]
@mikenikles ?

**Comment by mikenikles** [comment]
Hey, yes you can create a static site with `sapper export` (https://sapper.svelte.dev/docs#sapper_export). I do this for my personal website for example.

You can definitely get rid of Deno and deploy your static site to Vercel.com, Firebase Hosting or any provider that supports static hosting, even GitHub Pages.

I created the Deno proof of concept when Deno 1.0 was released and I was curious to learn more about it.
